src/hmem_cuda.c

```diff
@@ -760,6 +760,12 @@ bool cuda_is_gdrcopy_enabled(void)
 	return hmem_cuda_use_gdrcopy;
 }
 
+int cuda_flush_rdma_operations(void) {
+	enum cudaFlushGPUDirectRDMAWritesTarget target = cudaFlushGPUDirectRDMAWritesTargetCurrentDevice;
+	enum cudaFlushGPUDirectRDMAWritesScope scope = cudaFlushGPUDirectRDMAWritesToOwner;
+	return cudaDeviceFlushGPUDirectRDMAWrites(target, scope);
+}
```

Comment:
Please fix formatting.  Opening brace for function should be on line by itself.  Limit line lengths to about 80 characters.  target and scope variables don't even look needed.

Reply:
fixed